src/test/tv/codely/scala_intro_examples/lesson_08_collections/WordCounterSpec.scala

```diff
@@ -10,13 +10,18 @@ final class WordCounterSpec extends WordSpec with Matchers {
 
       counter.countWords(emptyText) shouldBe 0
     }
-    // @ToDo: Here you have your exercise!
-    // Change this "ignore" keyword for the "in" one and make this test pass!
-    "return the number of words given a non empty text" ignore {
+    "return the number of words given a non empty text" in {
       val counter = new WordCounter
       val nonEmptyText = "Hi all"
       val wordsNumber = 2
 
+      counter.countWords(nonEmptyText) shouldBe wordsNumber
+    }
+    "return the number of words given a non empty text with spaces" in {
```

Comment:
Good one!

Suggestion: Add other corner cases such as empty strings (0), punctuation, and so on 😬